svelte-kit-scss/src/lib/components/RepositoryCard/RepositoryCard.spec.ts

```diff
@@ -35,8 +36,8 @@ describe('RepositoryCard', () => {
     expect(license).toBeTruthy();
   });
 
-  // it('should should render repo last update', () => {
-  //   const lastUpdate = screen.getByText(/weeks ago/);
-  //   expect(lastUpdate).toBeTruthy();
-  // });
+  it('should should render repo last update', () => {
+    const lastUpdate = screen.getByText(relativeTimeFmt('2022-10-25T16:41:51Z'));
```

Comment:
What was the problem here?

It seems to me like the original text was "correct" in that it was expecting a certain string to appear based on the fixtures, but now we are using an implementation detail in the test, to assert the text shown.

Reply:
The relative time was changing breaking the test. Putting a hard coded string will not work in this  case

Reply:
I see.
So I understand then, the fixtures have a set time, which then displays in the component as relative to "now", which of course changes whenever the test is run.

This is a tricky but common problem.

I think since the `relativeTimeFmt` is a direct dependancy of the `RepositoryCard`, that this use in the test is appropriate, an alternative would've been to mock `relativeTimeFmt` which seems excessive, so that's just a long winded way of saying, this is good.